lib/organisms/gravity-form/gravity-form.controller.js

```diff
@@ -1,169 +1,151 @@
-(function () {
-
-    /**
-     * Created by Chris on 12/04/2016.
-     */
-
-    'use strict';
-
-
-    var FORM_INPUTS = [
-        'input', 'textarea', 'select', 'datalist', 'keygen'
-    ];
-
-
-    function lnOGravityFormController($log, $scope, $element, $attrs, $sce, lnOGravityFormService) {
-
-        var
-            ctrl = this,
-            inputs = undefined
-        ;
+angular
+    .module('lnPatterns')
+    .controller('lnOGravityFormController', lnOGravityFormController);
+
+/*@ngInject*/
+function lnOGravityFormController($log, $scope, $element, $attrs, $sce, lnOGravityFormService) {
+
+    var
+        FORM_INPUTS = [
+            'input', 'textarea', 'select', 'datalist', 'keygen'
+        ],
+        ctrl = this,
+        inputs = undefined
+    ;
 
-        this.submitDisabled = false;
+    this.form = $element[0];
```

Comment:
We use the `vm` notation for controllers - eg 

```
  var vm = this;
    vm.name = {};
````

See https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y032